packages/x-codemod/README.md

```diff
@@ -220,6 +221,56 @@ Replace props used for `Tabs` in DateTime pickers by `componentsProps.tabs` prop
 npx @mui/x-codemod v6.0.0/pickers/replace-tabs-props <path>
 ```
 
+#### `migrate-to-components-componentsProps`
+
+Replace customization props by their equivalent `components` adn `componentsProps` properties.
+
+```diff
+ <DatePicker
+-  PopperProps={{ onClick: handleClick }}
++  componentsProps={{ popper: { onClick: handleClick }}}
+ />
+
+ <DatePicker
+-  TransitionComponent={Fade}
++  components={{ DesktopTransition: Fade }}
+ />
+
+ <DatePicker
+-  DialogProps={{ backgroundColor: 'red' }}
++  componentsProps={{ dialog: { backgroundColor: 'red }}}
+ />
+
+ <DatePicker
+-  PaperProps={{ backgroundColor: 'red' }}
++  componentsProps={{ desktopPaper: { backgroundColor: 'red }}}
+ />
+
+ <DatePicker
+-  TrapFocusProps={{ isEnabled: () => false }}
++  componentsProps={{ desktopTrapFocus: { isEnabled: () => false }}}
+ />
+
+ <DatePicker
+-  InputProps={{ color: 'primary' }}
++  componentsProps={{ input: { InputProps: { color: 'primary' }}}}
```

Comment:
This is technically incorrect, because of https://github.com/mui/mui-x/pull/7369 😭 
What do you think:
- should we update existing migration entries to match this change?
- create a new entry and possibly a separate codemod?
I'm leaning towards updating the existing entry, because it's a continuation of a previous breaking change. 🤔

Reply:
Yes, we can replace it by
```
componentsProps={{ textField: { InputProps: { color: 'primary' }}}}
```

People following the alpha might already fix it.

Reply:
Could you please update the migration doc with the same diff as well?
We missed it in the original PR... 🙈